packages/components/src/components/modal/modal-styles.ts

```diff
@@ -160,9 +161,9 @@ export const getComponentCss = (
         color: primaryColor, // enables color inheritance for slotted content
         position: 'relative',
         boxSizing: 'border-box',
-        transform: isOpen ? 'scale3d(1,1,1)' : 'scale3d(.9,.9,1)',
+        transform: isOpen ? 'translateY(0%)' : 'translateY(25%)',
```

Comment:
Scaling is gone now?

Reply:
Yes, scaling is gone now.

Reply:
Hm.. if that was an option a few weeks, the entire sticky footer stuff could have been solved with intersection observers instead of scroll listeners 🤷‍♂️